config/config_helpers_test.go

```diff
@@ -138,11 +138,17 @@ func TestRunCommand(t *testing.T) {
 		expectedErr       error
 	}{
 		{
-			`"/bin/bash", "-c", ""echo -n foo""`,
+			`"/bin/bash", "-c", "echo -n foo"`,
```

Comment:
This test doesn't make much sense.

By splitting the arguments into an array, there's no need to rely on quoting to handle spaces.  This is something we do on the shell command-line _because_ the string hasn't already been split.

This means that `"echo -n foo"` is already treated as a single parameter being passed to `bash`.  There's no need to quote it again.

In fact, if we do, we're actually saying that we want to pass the string `"echo -n foo"` to the bash interpreter -- which is of course an invalid command.

It just so happened to work in the old implementation because there is a unconditional removal of all quotes.  With proper(ish) escape handling, this is no longer a valid test case.

Hopefully, no one was relying on this behavior.